http/task_service.go

```diff
@@ -1580,6 +1591,59 @@ func (t TaskService) FindRuns(ctx context.Context, filter influxdb.RunFilter) ([
 		return nil, 0, err
 	}
 
+	if filter.BeforeTime != "" || filter.AfterTime != "" {
```

Comment:
Can you provide some more explanation of the goal of this new block of code?

Reply:
This is the implementation that handles when a user enters a particular time using the --after or --before flags. It handles the case where only the -after flag is used, only the -before flag is used, and where both flags are used.